### ISSUE TITLE
MatMenu does not focus items correctly when opened by mouse

### ISSUE THREAD

**Issue by chrisbubernak** [angular/material2#10980]
#### Bug, feature request, or proposal:

Repro:
1) go to https://material.angular.io/components/menu/overview
2) open the basic menu via mouse (at this point no item should be visibly focused)
3) use arrow keys

#### What is the expected behavior?

items would get visibly focused (i.e gray highlighting)

#### What is the current behavior?

items don't get focused

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>
